integration/__tests__/cluster-pages.tests.ts

```diff
@@ -377,10 +377,8 @@ utils.describeIf(minikubeReady(TEST_NAMESPACE))("Minikube based tests", () => {
 
     await kubeApiServerRow.click();
     await frame.waitForSelector(".Drawer", { state: "visible" });
-
-    const logButton = await frame.waitForSelector("ul.KubeObjectMenu li.MenuItem i.Icon span[data-icon-name='subject']");
-
-    await logButton.click();
+    await frame.waitForSelector(".Drawer .drawer-title .Icon >> text=subject").then(button => button.click());
```

Comment:
nit: `await frame.click(...);`

Reply:
Did you mean `(await frame.waitForSelector(".Drawer .drawer-title .Icon >> text=subject")).click()`?